app/controllers/application_controller.rb

```diff
@@ -111,7 +111,7 @@ def render_not_found
       format.html { render file: Rails.public_path.join("404.html"), status: :not_found, layout: false }
       format.json { render json: { error: t(:not_found) }, status: :not_found }
       format.yaml { render yaml: { error: t(:not_found) }, status: :not_found }
-      format.any(:all) { render text: t(:not_found), status: :not_found }
+      format.any(:all) { render plain: t(:not_found), status: :not_found }
```

Comment:
`render text:` was removed in Rails 5.1 https://github.com/rails/rails/commit/79a5ea9eadb4d43b62afacedc0706cbe88c54496